core/io/resource_saver.cpp

```diff
@@ -41,9 +41,17 @@ bool ResourceSaver::timestamp_on_save = false;
 ResourceSavedCallback ResourceSaver::save_callback = nullptr;
 ResourceSaverGetResourceIDForPath ResourceSaver::save_get_id_for_path = nullptr;
 
-Error ResourceFormatSaver::save(const Ref<Resource> &p_resource, const String &p_path, uint32_t p_flags) {
+Error ResourceFormatSaver::save(const Ref<Resource> &p_resource, const String &p_path, uint32_t p_flags, FileAccess::SaveIntegrityLevel p_integrity_level) {
 	Error err = ERR_METHOD_NOT_FOUND;
-	GDVIRTUAL_CALL(_save, p_resource, p_path, p_flags, err);
+	if (GDVIRTUAL_CALL(_save, p_resource, p_path, p_flags, p_integrity_level, err)) {
+		return err;
+	}
+
+#ifndef DISABLE_DEPRECATED
+	// If not, we fall back on the old one. Notice the "_save_compat_100447" alias we're using here instead of "_save".
+	GDVIRTUAL_CALL(_save_compat_100447, p_resource, p_path, p_flags, err);
```

Comment:
what is going on here? `_save_compat_100447` [only takes a `path` argument](https://github.com/godotengine/godot/pull/100447/files#diff-3b9dd8590c4e9192c964dc726efa3f1a3911d6956a7902fa19e90453f8cef9edR52) but i see 4 arguments here, including a non reference or pointer `err` argument.
How does this work?

Reply:
You were looking at `ConfigFile::_save_compat_100447`; but you should be looking at `ResourceFormatSaver::_save_compat_100447` which takes a `path` argument and a `flags` argument. 2 arguments in total.

There are other issues with `ResourceFormatSaver::_save_compat_100447`, I'll be pushing a fix in 10 or 20 minutes.

Reply:
OK all issues I could find are fixed. CI is till running though.